### HYPOTHESIS TESTING

#### Assignment Tasks:

1. State the Hypotheses statement:

Null Hypothesis:
* H0: The weekly operating costs is not higher than $1,000 + $5X
* H0: W<=$1,000 + $5X

Alternative Hypothesis:
* H1: The weekly operating costs is higher than $1,000 + $5X
* H1: W>$1,000 + $5X

2. Calculate the Test Statistic:

Data Given:
* x_bar = sample mean weekly cost = Rs. 3,050
* μ = theoretical mean weekly cost according to the cost model (W = $1,000 + $5X for X = 600 units)
* σ = 5*25 units
* n = sample size (25 restaurants)

In [4]:
import numpy as np
import scipy.stats as stats
import statsmodels
import warnings
warnings.filterwarnings('ignore')

In [5]:
x_bar=3050 #sample mean
print('sample mean:',x_bar)

# μ=1000+5(600) where x=600 μ=4000
theo_mean_cost=4000
print('theoretical mean cost:',theo_mean_cost)

# σ = 5*25 units=125 #standard deviation
std_dev=125
print('Standard deviation of cost:',std_dev)

#sample size
n=25
print('sample size:',n)

sample mean: 3050
theoretical mean cost: 4000
Standard deviation of cost: 125
sample size: 25


In [6]:
#calculating the test statistic(z)
zscore= (x_bar-theo_mean_cost)/(std_dev/np.sqrt(n))
print('test statistic(z):',zscore)

test statistic(z): -38.0


3. Determine the Probability and compare:
Using the alpha level of 5% (α = 0.05),


In [7]:
alpha=0.05

We need to calculate z critical now using alpha as 5%. Since alternate hypothesis says that weekly costs are higher than W, it is a one tailed right tailed test.

In [8]:
# calculating the critical value
z_critical=stats.norm.ppf(0.95)
print('critical value:',z_critical)

critical value: 1.6448536269514722


4. Make a Decision:
Compare the test statistic with the critical value to decide whether to reject the null hypothesis.

* zscore < z_critical (-38 < 1.64)
* Therefore, We fail to reject H0.
* Decision: The weekly costs is not higher than W=$1000+5X.

5. Conclusion:

Based on the decision in step 4, conclude whether there is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.


Answer: Based on the decision in step 4, there is no strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests. The evidence shows that the cost is lesser thanwhat the model suggests.

The calculations clearly show that the critical value is larger than the test statistic which leads to failure to reject H0. That is, the restautant owners claim (alternative hypothesis) is proved to be wrong.